django/contrib/admin/static/admin/css/base.css

```diff
@@ -2,60 +2,89 @@
     DJANGO Admin styles
 */
 
-@import url(fonts.css);
-
 /* VARIABLE DEFINITIONS */
 html[data-theme="light"],
 :root {
-  --primary: #79aec8;
```

Comment:
I fixed the indentation here as it's different from the rest of the file, but I can revert this or put in another commit maybe.

Reply:
Yes please. (See other comment 🙂)

Reply:
I moved to a separate commit as below.